samples/sql-persistence/simple/SqlPersistence_2/EndpointOracle/Program.cs

```diff
@@ -0,0 +1,58 @@
+﻿using System;
+using System.Threading.Tasks;
+using NServiceBus;
+using NServiceBus.Persistence.Sql;
+using Oracle.ManagedDataAccess.Client;
+
+internal class Program
+{
+    static void Main()
+    {
+        AsyncMain().GetAwaiter().GetResult();
+    }
+
+    static async Task AsyncMain()
+    {
+        Console.Title = "EndpointOracle";
+
+        #region OracleConfig
+
+        var endpointConfiguration = new EndpointConfiguration("EndpointOracle");
+
+        var persistence = endpointConfiguration.UsePersistence<SqlPersistence>();
+        var password = Environment.GetEnvironmentVariable("OraclePassword");
+        if (string.IsNullOrWhiteSpace(password))
+        {
+            throw new Exception("Could not extract 'OraclePassword' from Environment variables.");
+        }
+        var username = Environment.GetEnvironmentVariable("OracleUserName");
+        if (string.IsNullOrWhiteSpace(username))
+        {
+            throw new Exception("Could not extract 'OracleUserName' from Environment variables.");
+        }
+        var connection = $"Data Source=(DESCRIPTION=(ADDRESS_LIST = (ADDRESS = (PROTOCOL = TCP)(HOST = localhost)(PORT = 1521)))(CONNECT_DATA = (SERVER = DEDICATED)(SERVICE_NAME = XE)));User Id={username}; Password={password}; Enlist=dynamic";
```

Comment:
Woah, do we need all this? Also main doco says (because this is what our acceptance tests do) to use Enlist=false.